cancellation/src/workflows/example.ts

```diff
@@ -12,9 +13,9 @@ const activityToBeCancelledWithTimeout = Context.configure(activityToBeCancelled
 async function main() {
   let err;
   try {
-    await activityToBeCancelledWithTimeout();
+    await activityToBeCancelled();
   } catch (_err) {
-    if (!(_err instanceof CancelledError)) {
+    if (!(_err instanceof CancelledFailure)) {
```

Comment:
This won't be `CancelledFailure`, use `isCancellation` instead: https://docs.temporal.io/docs/node/cancellation-scopes#cancelledfailure